src/objects.js

```diff
@@ -1,41 +1,49 @@
 const createPerson = (name, age) => {
-  // your code here
+  return {name, age};
 };
 
 const getName = object => {
-  // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  if (object[property]) return true;
+  return false;
 };
 
 const isOver65 = person => {
-  // your code here
+  if (person.age > 65) return true;
+  return false;
 };
 
 const getAges = people => {
-  // your code here
+  return people.map(newArray => newArray.age);
 };
 
 const findByName = (name, people) => {
-  // your code here
+  return people.find(thePerson => thePerson.name === name);
 };
 
 const findHondas = cars => {
-  // your code here
+  return cars.filter(theCars => theCars.manufacturer === "Honda");
```

Comment:
This is almost perfect. We are only talking about a single `car` though within the filter function. The plural could confuse people reading the code. So `return cars.filter(car => car.manufacturer === "Honda");` would be better :)